www/front_src/src/Navigation/Sidebar/Menu/index.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-param-reassign */
```

Comment:
I would'nt disable this rule as it is an important one, I would rather find a way to implement the addNavigationItemSelected function by not mutating